File: app/src/main/java/com/mammoth/soft/huarongdao/utils/AppDatabase.java
package com.mammoth.soft.huarongdao.utils;

import androidx.room.Database;
import androidx.room.RoomDatabase;

/**
 * Created by wuhaoyong on 2/03/20.
 */
@Database(entities = {GameHRD.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract GameHRDDao gameHRDDao();
}

File: app/src/main/java/com/mammoth/soft/huarongdao/data/AppDatabase.kt
package com.mammoth.soft.huarongdao.data

import androidx.room.Database
import com.mammoth.soft.huarongdao.data.GameHRD
import androidx.room.RoomDatabase
import com.mammoth.soft.huarongdao.data.GameHRDDao

/**
 * Created by wuhaoyong on 2/03/20.
 */
@Database(entities = arrayOf(GameHRD::class), version = 1)
abstract class AppDatabase : RoomDatabase() {
    abstract fun gameHRDDao(): GameHRDDao?
}